# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/tests/browser/tree/browser.toml
# Commit: 1fbca7956efd
# Full Hash: 1fbca7956efd1b73ae3cc50f21ac2780a9e2040c
# Author: Morgan Rae Reschenberg <mreschenberg@mozilla.com>
# Date: 2025-04-10 09:29:04
# Regressor Bug: 1549202
# File Overlap Count: 1
# Description:
#   Bug 1549202: Ensure ContentSelectDropdown local accessibles return the focused <select> remote accessible as their parent r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D243029
# ==============================================================================

diff -r 0b9e529a5541 -r 1fbca7956efd accessible/tests/browser/tree/browser.toml
--- a/accessible/tests/browser/tree/browser.toml	Wed Apr 09 23:51:01 2025 +0000
+++ b/accessible/tests/browser/tree/browser.toml	Wed Apr 09 23:51:27 2025 +0000
@@ -30,6 +30,8 @@
 
 ["browser_searchbar.js"]
 
+["browser_select.js"]
+
 ["browser_shadowdom.js"]
 
 ["browser_test_nsIAccessibleDocument_URL.js"]